OpenRA.Mods.Cnc/Activities/Leap.cs

```diff
@@ -113,7 +115,9 @@ public override Activity Tick(Actor self)
 				return this;
 			}
 
-			mobile.IsMoving = true;
+			wasMoving = mobile.IsMoving;
+			if (!wasMoving)
+				mobile.IsMoving = true;
```

Comment:
This shouldn't be needed - the `IsMoving` setter already checks the previous value and only notifies consumers if it changed.